build.gradle

```diff
@@ -256,6 +256,8 @@ allprojects {
                 '**/returnsreceiverdelomboked/*',
                 '**/build/**',
                 '*/dist/**',
+                // Don't format util the error report location is fixed
```

Comment:
All tests should be formatted properly. There is nothing special about PICO tests.

Reply:
Yes, that's the next thing we should look at in the future. I opened an issue in opprop https://github.com/opprop/immutability/issues/53.